.env.production

```diff
@@ -2,7 +2,6 @@
 NODE_ENV=production
 BABEL_ENV=production
 VUE_APP_ENVIRONMENT="production"
-VUE_APP_PUBLIC_PATH="/qvainpilot/"
```

Comment:
```
$ grep -nir "VUE_APP_PUBLIC_PATH" .
./.env.production:5:VUE_APP_PUBLIC_PATH="/qvainpilot/"
./.env:1:VUE_APP_PUBLIC_PATH='/'
./vue.config.js:5:	publicPath: process.env.VUE_APP_PUBLIC_PATH,
./src/router.js:46:	base: process.env.VUE_APP_PUBLIC_PATH,
```

Should we have that VUE_APP_PUBLIC_PATH='/' value in .env.production?

Reply:
It's not necessary. If we don't override the value in .env.production, the value from .env is used.